go/roothash/api/commitment/pool.go

```diff
@@ -263,6 +263,8 @@ func (p *Pool) addOpenExecutorCommitment(
 		return ErrTxnSchedSigInvalid
 	}
 
+	// TODO: check for evidence of equivocation.
```

Comment:
shall we file this?

Reply:
Filed https://github.com/oasisprotocol/oasis-core/issues/3685.